### ISSUE TITLE
Wintergrasp cannons dont award rank when destroyed

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5945]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
When destroying a tower/wall cannon in WG it doesnt award rank like npcs do
**Expected behaviour**: Tell us what should happen instead
After killing a tower/wall cannon you should gain rank just as you do from npcs
**Steps to reproduce the problem**:

1. destroy a cannon
2. see how it didnt increase rank
3. 

**Include proofs for this behaviour**
https://wowwiki.fandom.com/wiki/Wintergrasp
you can read at section "Ranks" that you should gain rank after killing a cannon
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
